spec/models/content_page_spec.rb

```diff
@@ -43,4 +47,49 @@
     expect(saved_page.slug.count("-")).to be(3)
     expect(saved_page.title.count("-")).to be(2)
   end
+
+  # The order is a depth first search
+  context "Next and Previous pages" do
```

Comment:
It would be nice to see this test split into two, one block to test the Next unit and another to test the Previous unit.
By testing the methods in isolation leads to less coupling down the line

Reply:
Done